packages/edit-post/src/components/welcome-guide/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __experimentalGuide as Guide } from '@wordpress/nux';
+import { __ } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { CanvasImage, EditorImage, BlockLibraryImage, InserterIconImage } from './images';
+
+export default function WelcomeGuide() {
+	const areTipsEnabled = useSelect( ( select ) => select( 'core/nux' ).areTipsEnabled() );
+
+	const { disableTips } = useDispatch( 'core/nux' );
+
+	if ( ! areTipsEnabled ) {
+		return null;
+	}
+
+	return (
+		<Guide
+			className="edit-post-welcome-guide"
+			finishButtonText={ __( 'Get started' ) }
+			onFinish={ disableTips }
+		>
+
+			<Guide.Page className="edit-post-welcome-guide__page">
```

Comment:
`Guide.Page` this is not a pattern we used that much, we tend to favor `GuidePage` instead. (PanelBody, ...) 
I'm not strongly against it but we should be consistent I think.

Reply:
👌 ec96e0e7af8c2b82be8174bd0f4d2fe5b9c8afbc